client/my-sites/plans-v2/utils.ts

```diff
@@ -232,6 +236,9 @@ export function itemToSelectorProduct(
 			shortName: getJetpackProductShortName( item ) || '',
 			tagline: getJetpackProductTagline( item ),
 			description: getJetpackProductDescription( item ),
+			children: JETPACK_SEARCH_PRODUCTS.includes( item.product_slug )
```

Comment:
This might need to be extracted in a separate file or function. Search is the only product that uses a children, though.